google-cloud-redis/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+### 0.3.0 / 2019-04-29
+
+* Add Instance#persistence_iam_identity attribute.
+* Add CloudRedisClient#import_instance.
+* Add CloudRedisClient#export_instance.
```

Comment:
Weren't the features for these two lines removed?

Reply:
Good catch, thanks! Removed these lines.